### PR TITLE
OSDOCS#5138: Adding grep -v guard to restoring cluster commands

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.10+

Issue:
https://issues.redhat.com/browse/OSDOCS-5138

Link to docs preview:
https://59826--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -85,7 +85,7 @@ $ sudo mv /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
 +
 [source,terminal]
 ----
-$ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard"
+$ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard" | grep -v guard
```
Comment (kenneth-dsouza) [inline modules/dr-restoring-cluster-state.adoc:88]: Not sue if we need to pass grep -v guard again ? As we are covering that in egrep.

File: modules/dr-restoring-cluster-state.adoc [hunk 2]
```diff
@@ -256,7 +256,7 @@ $ oc adm certificate approve <csr_name>
 +
 [source,terminal]
 ----
-$ sudo crictl ps | grep etcd | grep -v operator
+$ sudo crictl ps | grep etcd | grep -v operator | grep -v guard
```
Comment (kenneth-dsouza) [inline modules/dr-restoring-cluster-state.adoc:259]: would the below do? As we are using the same https://github.com/openshift/openshift-docs/pull/55879/files

` sudo crictl ps | grep etcd | egrep -v "operator|etcd-guard" `

File: modules/dr-restoring-cluster-state.adoc (hunk 1 above)
Comment (obrown1205) [reply modules/dr-restoring-cluster-state.adoc:88]: Ah, ok! Will remove

Comment (obrown1205) [reply modules/dr-restoring-cluster-state.adoc]: Updated :)

File: modules/dr-restoring-cluster-state.adoc (hunk 2 above)
Comment (obrown1205) [reply modules/dr-restoring-cluster-state.adoc:259]: Will adjust!

**Comment by obrown1205** [comment]
@geliu2016 - would you mind taking a look? Tysm!

**Comment by geliu2016** [comment]
/lgtm
Tried it, the new command line output will ignore "guard" container.

**Review by lpettyjo — COMMENTED** [review]
LGTM

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM

**Pull request merged**